src/components/Content/index.tsx

```diff
@@ -37,7 +38,8 @@ export const Content: React.FC = () => {
       <Switch>
         <PrivateRoute path={routes.task} component={Task} />
         <PrivateRoute path={routes.configurePatch} component={ConfigurePatch} />
-        <PrivateRoute path={routes.patch} component={Patch} />
+        <PrivateRoute path={routes.patch} component={PatchRedirect} />
+        <PrivateRoute path={routes.version} component={Patch} />
```

Comment:
Now that `Patch` is no longer a page I think that changing the component name to `Version` is the right move